vnext/ReactUWP/Polyester/IconViewManager.cpp

```diff
@@ -65,9 +65,11 @@ void IconShadowNode::updateProperties(const folly::dynamic &&props) {
     if (propertyName == "color") {
       if (IsValidColorValue(propertyValue))
         glyphs.Fill(BrushFrom(propertyValue));
-#if FUTURE
-      else if (propertyValue.isNull())
-        ; // Log error, must have a color
+#ifdef DEBUG
+      else if (propertyValue.isNull()) {
+        // Log error, must have a color
+        YellowBox("IconShadowNode - color property must be non-null");
```

Comment:
Generally Facebook only does this warning once per instance.  You wouldn't want your app spammed with this error.
Unfortunately their warnOnce API isn't exposed to native.  But then again, its probably not hard to do something on the native side to only do this once.

Reply:
YellowBox _already_ aggregates/coalesces like warnings into one and you just get one warning per message, with a badge for the # of times it has hit.
![image](https://user-images.githubusercontent.com/22989529/80626332-234ceb80-8a03-11ea-9fea-ad0dbc98507d.png)

Reply:
I believe that it only does that if it matches the last error.  So once we have multiple of these it would get very spammy.

Reply:
@acoates-ms confirmed that when I get the 53 test ones, then go to another page that shows a yellow box, then go to the 53 test yellowbox, the test yellowbox correctly adds those up and brings it to the top:

![image](https://user-images.githubusercontent.com/22989529/80633369-7e83db80-8a0d-11ea-8065-98823b3ec990.png)